src/oscar/apps/customer/migrations/0004_auto_20170413_1853.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-04-13 17:53
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('customer', '0003_update_email_length'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
Not quite:

- add optional field `email` (0004_auto_20170413_1853.py);
- fill in `email` field with data taken from `User.email` field (0005_auto_20170413_1857.py);
- make `email` field mandatory since data is in place (0006_auto_20170413_1901.py).